### ISSUE TITLE
WangId documentation is incorrect

### ISSUE THREAD

**Issue by Phlosioneer** [bjorn/tiled#2737]
The documentation says:
> wangid: The Wang ID, which is a 32-bit unsigned integer stored in the format `0xCECECECE` (where each C is a corner color and each E is an edge color, from right to left clockwise, starting with the top edge)

This is my test case: 
![testCase](https://user-images.githubusercontent.com/4657718/73154544-96ae2c80-40a5-11ea-9371-836197552637.png)

The TSX output is:
```xml
<wangset name="test" tile="-1">
   <wangedgecolor name="" color="#ff0000" tile="-1" probability="1"/>
   <wangedgecolor name="" color="#00ff00" tile="-1" probability="1"/>
   <wangedgecolor name="" color="#0000ff" tile="-1" probability="1"/>
   <wangedgecolor name="" color="#ff7700" tile="-1" probability="1"/>
   <wangcornercolor name="" color="#ff0000" tile="-1" probability="1"/>
   <wangcornercolor name="" color="#00ff00" tile="-1" probability="1"/>
   <wangcornercolor name="" color="#0000ff" tile="-1" probability="1"/>
   <wangcornercolor name="" color="#ff7700" tile="-1" probability="1"/>
   <wangtile tileid="430" wangid="0x14433221"/>
  </wangset>
```

So it seems to be formatted `0xCECECECE`, starting with the upper left corner, and going counter-clockwise. None of that lines up with any part of the documentation. Am I misreading the docs? Or misreading the test case?

**Comment by bjorn** [comment]
> The Wang ID, which is a 32-bit unsigned integer stored in the format `0xCECECECE` (where each C is a corner color and each E is an edge color, from right to left clockwise, starting with the top edge)

It lines up if you consider the "starting with the top edge" to refer to the `E` on the right. I agree this is not exactly intuitive, so this should be made more explicit.

**Comment by bjorn** [comment]
This was addressed as part of 2749ba1fdbf664e09ef97e026719588a9ac1a3ca.
